Examples/Swift/ManageOfflineRegionsExample.swift

```diff
@@ -1,220 +1,234 @@
+    override func viewDidLoad() {
+
+        super.viewDidLoad()
+
+        view.addSubview(mapView)
+        view.addSubview(tableView)
+        mapView.addSubview(downloadButton)
+        mapView.setCenter(CLLocationCoordinate2D(latitude: 22.27933, longitude: 114.16281),
+
+                          zoomLevel: 13, animated: false)
+
+        // Setup offline pack notification handlers.
+        addObserver()
+
+        if #available(iOS 11.0, *) {
             installConstraints()
+        } else {
+            // Fallback on earlier versions
         }
+    }
 
-        func installConstraints() {
-
-            if #available(iOS 11.0, *) {
-                NSLayoutConstraint.activate([
-                    downloadButton.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
-                    downloadButton.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -170),
-                    downloadButton.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor)
-                ])
-            } else {
-                // Fallback on earlier versions
-            }
-
-        }
-
-        override func viewDidDisappear(_ animated: Bool) {
+    func addObserver() {
+        NotificationCenter.default.addObserver(self, selector: #selector(offlinePackProgressDidChange), name: NSNotification.Name.MGLOfflinePackProgressChanged, object: nil)
+    }
 
-            super.viewDidDisappear(animated)
-            // When leaving this view controller, suspend offline downloads.
-            guard let packs = MGLOfflineStorage.shared.packs else { return }
-            for pack in packs {
-                if let userInfo = NSKeyedUnarchiver.unarchiveObject(with: pack.context) as? [String: String] {
-                    print("Suspending download of offline pack: “\(userInfo["name"] ?? "unknown")”")
-                }
-                pack.suspend()
-            }
-        }
+    @available(iOS 11.0, *)
+    func installConstraints() {
+
+        NSLayoutConstraint.activate([
+            mapView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            mapView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+            mapView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+            mapView.heightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.heightAnchor, multiplier: 0.75),
+            tableView.topAnchor.constraint(equalTo: mapView.bottomAnchor),
+            tableView.leadingAnchor.constraint(equalTo: mapView.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: mapView.trailingAnchor),
+            tableView.heightAnchor.constraint(equalTo: mapView.heightAnchor, multiplier: 0.4),
+            downloadButton.topAnchor.constraint(equalTo: mapView.topAnchor, constant: 10),
+            downloadButton.leadingAnchor.constraint(equalTo: mapView.leadingAnchor),
```

Comment:
Can we add some extra space here? Looks a little too close to the edge:

![image](https://user-images.githubusercontent.com/10850812/96941790-41624880-1488-11eb-95de-d8b51766fb6b.png)

I'd also consider making the button a contrasting color with some bold text so it stands out from the map view's color scheme. Maybe try one of the [system colors](https://developer.apple.com/documentation/uikit/uicolor/standard_colors)?